project/Dependencies.scala

```diff
@@ -2,13 +2,20 @@ import sbt._
 
 object Dependencies {
 
-  val circeVersion   = "0.14.1"
-  val doobieVersion  = "0.13.4"
-  val fs2Version     = "2.5.6"
-  val h2Version      = "1.4.200"
-  val log4sVersion   = "1.8.2"
-  val logbackVersion = "1.2.3"
-  val munitVersion   = "0.7.26"
+  val akkaVersion      = "2.6.15"
+  val circeVersion     = "0.14.1"
+  val doobieVersion    = "0.13.4"
+  val fs2Version       = "2.5.6"
+  val h2Version        = "1.4.200"
+  val log4sVersion     = "1.8.2"
+  val logbackVersion   = "1.2.3"
+  val munitVersion     = "0.7.26"
+  val scalaTestVersion = "3.2.9"
```

Comment:
Is it really necessary to use scalatest? There is already munit in use. I'd prefer not to have multiple test frameworks. Is this required, because akka tests don't run without scala test?

Reply:
Yes, akka-testkit requires scala-test (the test extends from akka.actor.testkit.typed.scaladsl.ScalaTestWithActorTestKit)

Reply:
Ok.